cmd/minishift/cmd/openshift/restart.go

```diff
@@ -33,7 +33,7 @@ import (
 var restartCmd = &cobra.Command{
 	Use:   "restart",
 	Short: "Restarts the OpenShift cluster.",
-	Long:  "Restarts the OpenShift cluster, keeping the VM alive.",
+	Long:  "Restarts the OpenShift cluster, and maintains the running state of the Minishift VM.",
```

Comment:
I'll go with @hferentschik suggestion since it keeps a positive voice and is more concise.